facdb/pipelines.py

```diff
@@ -426,6 +422,20 @@ def fbop_corrections(df: pd.DataFrame = None):
 @ParseAddress(raw_address_field="facilityaddress")
 @Prepare
 def fdny_firehouses(df: pd.DataFrame = None):
+    research = (
```

Comment:
Do we expect this to be static enough to hard-code? I assume firehouses don't change much.

Reply:
yes, the ones not geocoded are the same records